crates/biome_lsp/src/capabilities.rs

```diff
@@ -60,8 +61,14 @@ pub(crate) fn server_capabilities(capabilities: &ClientCapabilities) -> ServerCa
             CodeActionOptions {
                 code_action_kinds: Some(vec![
                     CodeActionKind::from("quickfix.biome"),
+                    // quickfix.suppressRule
+                    CodeActionKind::from(SUPPRESSION_ACTION_CATEGORY),
                     CodeActionKind::from("source.fixAll.biome"),
                     CodeActionKind::from("source.organizeImports.biome"),
+                    CodeActionKind::from("refactor.biome"),
+                    CodeActionKind::from("refactor.extract.biome"),
+                    CodeActionKind::from("refactor.inline.biome"),
+                    CodeActionKind::from("refactor.rewrite.biome"),
```

Comment:
Probably these code actions are not available at all? Happy to remove them in this case.